docs/source/install.rst

```diff
@@ -107,29 +104,28 @@ you may install the Jupyter Notebook using Python's package manager `pip`::
         yum groupinstall 'Development Tools'
         yum install python3-devel
 
-    (Use ``python`` instead of ``python3`` for legacy Python 2)
-
-
-Now, you may consider :ref:`installing kernels <installing-kernels>` and
-:ref:`next steps <next-steps>`.
+    (Use ``python`` instead of ``python3`` for legacy Python 2.)
 
+See :ref:`installing additional kernels <installing-kernels>` to use programming
+languages other than Python and :ref:`next steps <next-steps>` for running the
+Jupyter Notebook.
 
 .. _upgrading:
 
-Upgrading to Jupyter: Experienced Users
----------------------------------------
+Upgrading a Jupyter installation
+--------------------------------
 
 The Jupyter Notebook used to be called the IPython Notebook. If you are running
 an older version of the IPython Notebook (version 3 or earlier) you can use the
 following to upgrade to the latest version of the Jupyter Notebook.
 
-**If using 'pip'**::
+**If using pip**::
 
     pip install -U jupyter
 
 OR
 
-**If using Anaconda or `conda`**::
+**If using Anaconda or conda**::
```

Comment:
I think we can just say 'If using Anaconda'. It's not really an either/or thing - conda is part of Anaconda. And the people who have installed conda by [other means](http://conda.pydata.org/miniconda.html) are probably technical enough to recognise that this applies to them too.